huxley/www/js/lib/ServerAPI.js

```diff
@@ -34,6 +34,20 @@ var ServerAPI = {
   },
 
   /**
+   * Get a list of all assignments for the given school ID.
+   */
+  getCommitteeAssignments(committeeID) {
+    return _get(`/api/assignments/committee/${committeeID}`);
```

Comment:
(I haven't looked at the backend PR yet, but I'm just taking a guess based on the URI that it'll be a new API endpoint.